src/components/SecurityContainer/SecurityContainer.js

```diff
@@ -0,0 +1,108 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import Description from '../Description/Description'
+
+import './SecurityContainer.scss'
+
+export default class SecurityContainer extends PureComponent {
+  render () {
+    const { id, security } = this.props
+    const isSimple = ['apiKey', 'http'].includes(security.type)
+
+    return (
+      <section className='security-container' id={id}>
+        {isSimple && this.renderSimple(security)}
+        {security.type === 'oauth2' && this.renderOAuth2(security)}
+        {security.type === 'openIdConnect' && this.renderOpenIdConnect(security)}
```

Comment:
Looks like all 3 render methods share something in common:
```
   <h3>{name} <code className='scheme'>type={type}</code></h3>
   {description && <Description description={description} />}
```

How about moving that code to here?